### PR TITLE
diagram export toolbar command

### PR DESCRIPTION
**Pull request by romanresh**
adds the export button to the toolbar (supports svg, jpg and png export) and the "export" options

### CODE REVIEW

File: js/ui/diagram/ui.diagram.commands.js [hunk 1]
```diff
@@ -99,6 +102,33 @@ const DiagramCommands = {
                 displayExpr: "name",
                 valueExpr: "value"
             },
+            {
+                widget: "dxButton",
+                icon: "export",
+                items: [
+                    {
+                        command: DiagramCommand.ExportSvg, // eslint-disable-line spellcheck/spell-checker
+                        text: "Export to SVG",
```
Comment (maxerokh) [inline js/ui/diagram/ui.diagram.commands.js:111]: Possible, before release 19.1 you will should use DevExtreme's localization mechanism

File: js/ui/diagram/ui.diagram.js [hunk 2]
```diff
@@ -287,7 +288,26 @@ class Diagram extends Widget {
...
+
+            /**
+             * @name dxDiagramOptions.export
+             * @type object
+             */
+            "export": {
+                /**
+                 * @name dxDiagramOptions.export.fileName
+                 * @type string
+                 * @default "PivotGrid"
```
Comment (maxerokh) [inline js/ui/diagram/ui.diagram.js]: Default value is PivotGrid. Are you sure?

**Review by maxerokh — CHANGES_REQUESTED** [review]

File: js/ui/diagram/ui.diagram.js (hunk 2 above)
Comment (romanresh) [reply js/ui/diagram/ui.diagram.js]: mistake, thanks.

File: js/ui/diagram/ui.diagram.commands.js (hunk 1 above)
Comment (romanresh) [reply js/ui/diagram/ui.diagram.commands.js:111]: ok

**Review by maxerokh — APPROVED** [review]

**Pull request merged**